src/components/new-request/search.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-  <v-container fluid id="search-container" class="copy-normal pt-10 px-10"
-    :class="(isFederal && isRestoration) ? 'pb-0' : 'pb-12'"
-  >
+  <v-container fluid id="search-container" class="copy-normal pt-10 px-10 pb-12">
```

Comment:
Reverted this change that I made a in a PR a couple of days back.

Seems to whatever was causing the spacing issue here was collaterally fixed with all the refactoring that we've been doing 😁

Reply:
Hahaha. 😁

Reply:
🚀